stubs/bleach/bleach/html5lib_shim.pyi

```diff
@@ -1,30 +1,69 @@
-from _typeshed import Incomplete
-from collections.abc import Generator, Iterable, Iterator
+import re
+from codecs import CodecInfo
+from collections.abc import Collection, Generator, Iterable, Iterator
+from typing import Any
 
-class HTMLParser:  # actually html5lib.HTMLParser
-    def __getattr__(self, __name: str) -> Incomplete: ...
+# We don't re-export any `html5lib` types / values here, because they are not
+# really public and may change at any time. This is just a helper module,
+# import things directly from `html5lib` instead!
+from html5lib import HTMLParser
+from html5lib._inputstream import HTMLUnicodeInputStream
+from html5lib._tokenizer import HTMLTokenizer
+from html5lib._trie import Trie
+from html5lib.serializer import HTMLSerializer
+from html5lib.treewalkers.base import TreeWalker
 
-class Filter:  # actually html5lib.filters.base.Filter
-    source: Incomplete
-    def __init__(self, source) -> None: ...
-    def __iter__(self) -> Iterator[Incomplete]: ...
-    def __getattr__(self, name: str) -> Incomplete: ...  # copy attributes from source
+# Is actually webencodings.Encoding
+class _Encoding:
+    name: str
+    codec_info: CodecInfo
+    def __init__(self, name: str, codec_info: CodecInfo) -> None: ...
```

Comment:
I'd prefer to make this a protocol, since this isn't actually where the type is defined at runtime